app/Http/Controllers/FavoritesController.php

```diff
@@ -30,10 +30,10 @@ public function store(Request $request, Session $session): void
         }
     }
 
-    public function check($city_id): bool
+    public function check($cityId): bool
```

Comment:
Where is type?
Where is Request $request?

Reply:
Done